### PR TITLE
Recommended mode for operator upgrade updated 

### PR DESCRIPTION
**Pull request by shefali-malhotra**
# Description
Recommended mode for operator upgrade updated 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [X] Did you add the examples wherever applicable?
- [X] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — APPROVED** [review]

File: content/docs/deployment/csmoperator/_index.md [hunk 1]
```diff
@@ -350,6 +350,8 @@ The `Update approval` (**`InstallPlan`** in OLM terms) strategy plays a role whi
 
 >NOTE: The recommended version of OLM for Upstream Kubernetes is **`v0.25.0`**.
 
+>NOTE: The recommended Update Approval is **`Manual`** to prevent the inatsllation of non-qualified versions of operator.
```
Comment (bpjain2004) [inline content/docs/deployment/csmoperator/_index.md:353]: Can you update this to  
The recommended **`Update approval`** is

Comment (shefali-malhotra) [reply content/docs/deployment/csmoperator/_index.md]: addressed the review comment

**Review by rajendraindukuri — APPROVED** [review]

File: content/docs/deployment/csmoperator/_index.md [hunk 2]
```diff
@@ -350,6 +350,8 @@ The `Update approval` (**`InstallPlan`** in OLM terms) strategy plays a role whi
 
 >NOTE: The recommended version of OLM for Upstream Kubernetes is **`v0.25.0`**.
 
+>NOTE: The recommended **`Update Approval`** is **`Manual`** to prevent the inatsllation of non-qualified versions of operator.
```
Comment (rajkumar-palani) [inline content/docs/deployment/csmoperator/_index.md:353]: Can you please check the typo issue in the "installation"?

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
please correct the typo issues. thank you.

File: content/docs/deployment/csmoperator/_index.md (hunk 2 above)
Comment (shefali-malhotra) [reply content/docs/deployment/csmoperator/_index.md]: addressed

**Review by bpjain2004 — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**